oak/proto/policy.proto

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2019 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.policy;
+
+// Labels represents a policy associated with a message or a Node.
+//
+// See https://pdos.csail.mit.edu/papers/flume-sosp07.pdf section 3.1.
+message Labels {
+  repeated Tag secrecy_tags = 1;
+  // TODO: Support integrity tags.
+}
+
+// Tag represents a category of secrecy or integrity that is associated to a particular Node or
+// family of Nodes.
```

Comment:
I might have worded this differently &ndash; I think that tag is fundamentally attached to *data*, and because of the flow of that data, the tag also gets associated to Node(s) along the way,